packages/sources/src/sourceFormRenderer/components/CardSelect.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card, CardHeader, CardBody, FormGroup, Grid, GridItem, Bullseye } from '@patternfly/react-core';
+import { ServerIcon } from '@patternfly/react-icons';
+
+class CardSelect extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            selectedValues: this.props.input.value ? this.props.input.value : this.isMulti ? [] : undefined
+        };
+    }
+
+    isMulti = this.props.isMulti || this.props.multi;
+
+    isDisabled = this.props.isDisabled || this.props.isReadOnly;
+
+    change = () => this.props.input.onChange(this.state.selectedValues);
+
+    handleMulti = (value) => this.state.selectedValues.includes(value) ?
```

Comment:
I believe that selecting multiple sources was deprecated.

Reply:
It is for now. However, I am preparing for the future. :space_invader: